src/ducks/apps/components/AppInstallation.jsx

```diff
@@ -203,12 +205,16 @@ const mapStateToProps = (state, ownProps) => ({
   installError: state.apps.actionError
 })
 
-const mapDispatchToProps = dispatch => ({
+const mapDispatchToProps = (dispatch, ownProps) => ({
   installApp: (app, channel, isUpdate) =>
-    dispatch(installAppFromRegistry(app, channel, isUpdate))
+    dispatch(installAppFromRegistry(ownProps.client, app, channel, isUpdate))
 })
```

Comment:
We had to be careful using `ownProps`. Since it can brings a lot of re-render / re-calculation / dispatch if we have a prop update on the connected component. It can be a big drop of perf. 

Since I'm not familiar with this component, this is just a warning ;)